docs/faq.md

```diff
@@ -14,11 +14,11 @@ Currently this is a manual process. We are working on dashboards and other metri
 
 **Does MUO reserve compute capacity?**
 
-Yes, if `capacityReservation: true` is enabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.
+Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.
```

Comment:
If it is not specified at all when using a `local` provider, is it still disabled?

This might need a little rewording also because it reads a little bit inaccurately. MUO does not modify the count of existing worker machinesets. It creates a new `-upgrade` worker machineset for each availability zone with a size of 1.

Reply:
Thank you @mrbarge for review. I have updated the details for local provider in case `spec.capacityReservation` is missing in the upgradeconfig CR. Also  I have rephrased wording for `upgrade` worker machinesets.